README.md

```diff
@@ -17,17 +17,20 @@ Thank you for your interest in contributing to the Login.gov IdP! For complete i
 
 ### Installing on your local machine
 
-This installation method is meant for those who are familiar with setting up local development environments on their machines.
+This installation method is meant for those who are familiar with setting up local development environments on their machines. If you encounter errors, see the [Troubleshooting](#user-content-troubleshooting) section at the bottom of this README.
```

Comment:
It seems to work in previews as-is, but I think the anchor will be `#troubleshooting`:
```suggestion
This installation method is meant for those who are familiar with setting up local development environments on their machines. If you encounter errors, see the [Troubleshooting](#troubleshooting) section at the bottom of this README.
```

Reply:
This, I'm not so sure; if you see what github generates, it looks like it prefixes user-content-; check out the other headings on the README.